open-company/open-company-web
# Devcards improvements

card: https://trello.com/c/jXSzmjNf

to test:
- build and run figwheel and check the site
- build and run devcards
- open http://locahost:3449/devcards and check that the devcards are working
- navigate the devcards old revisions and check that the sections are not editable
- check the trello card for other changes
